pom.xml

```diff
@@ -229,6 +229,12 @@
             <version>${powsyblcore.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
@floriand-e2r Can you please check this. I am not sure about my modification.

Reply:
I'd rather use Supplier<T> (instead of Provider<T>) to avoid adding this dependency

Reply:
Do I have to make the code modification?